modin/engines/base/io/text/csv_reader.py

```diff
@@ -185,7 +185,10 @@ def _read(cls, filepath_or_buffer, **kwargs):
         # reported dtypes from differing rows can be different based on the inference in
         # the limited data seen by each worker. We use pandas to compute the exact dtype
         # over the whole column for each column. The index is set below.
-        dtypes = cls.get_dtypes(dtypes_ids)
+        if len(dtypes_ids) > 0:
+            dtypes = cls.get_dtypes(dtypes_ids)
+        else:
+            dtypes = None
```

Comment:
`get_dtypes` doesn't work with empty sequence?

Reply:
Yes, parameter `dtypes_ids` will be transferred to parameter `objs` of function `pandas.concat` internally. `pandas.concat` throws an error in case `objs=[]`.

Reply:
Is this `dtypes` defined for each partition? It turns out that we can have cases when the number of partitions is 0?

Reply:
Yes, we have cases when the number of partitions is 0 (when data is empty).

Reply:
got it.